app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,15 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
```

Comment:
- [ ] Please create a views/comments/new.html.erb file which will have a form that accepts input for creating comments.